docs/content/ingestion/firehose.md

```diff
@@ -169,3 +169,28 @@ when using above firehose the events can be sent via submitting a POST request t
 |type|receiver|yes|
 |serviceName|name used to announce the event receiver service endpoint|yes|
 |bufferSize| size of buffer used by firehose to store events|no default(100000)|
+
+#### KafkaSimpleConsumerFirehose (in experimental phase)
```

Comment:
can I also request some docs about using this firehose at scale and possibly in production?